contents/handbook/product/product-stages.md

```diff
@@ -0,0 +1,89 @@
+
+It should help address confusion around when to focus on feature parity vs innovative new major features. How much do we need to care about stability vs it being fine to move fast and break things. Should we keep investing in the smaller features or is doing the first 80% fine.
+
+**Caveat:** this guide mainly applies to building products where the category already exists. Defining a new category often requires different ordering/thinking.
+
+## The spectrum of product development
+
+Our products fall under three main categories within the product development spectrum:
+
+1. Minimum Viable Product (MVP)
+2. Product-Market Fit (PMF)
+3. Standalone Best-in-Class
+
+### 1. Minimum Viable Product (MVP)
+
+- Core features are in production; external rollout has started
+- Some rough edges may be present
+- Initially free to gather feedback and validate demand
+
+To reach the MVP stage:
+
+- Validate demand and identify crucial features through early customer conversations
+- Move fast, things might break, but fix issues quickly
+- Ruthlessly cut scope
+- Work with a small team (about 2 people)
+- Should move quickly based on gut/existing solutions in the market.
+
+When to go from an idea to an MVP:
+
+- TODO: add a link to the new section on which products we'll want to launch
+- Someone has conviction in the idea and envisions a rough path to PMF for our ICP
+- Take bets on ideas, understanding that some products may not reach PMF and could be deprecated
+
+### 2. Product-Market Fit (PMF)
+
+- Strong traction with Ideal Customer Profile (ICP) customers - we are commonly referenced in this category
+- Near feature parity with established tools
+- Differentiation mainly through integration with our other tools (If we can’t get to PMF through this then we consider doing bigger feature innovations)
+- Scalable, reliable, and profitable
+- API access for everything
+
+To progress from MVP to PMF:
+- Base decisions on customer feedback and market solutions
+- Prioritize features based on paying or potential paying customers' feedback
+- Focus on nailing the basics first
+- Begin charging once a user base is established
+- UX is familiar enough to existing solutions
+- Push hard, but recognize that some products may take time to reach PMF
+
+When should a product try to go from MVP to PMF:
+- Success for a product is reaching the PMF stage. We should only stop before that if the confidence in the product drops such as due to low uptake or high technical complexity
+
+### 3. Standalone Best-in-Class
+
+- Customers are signing up and paying for that specific tool
+- Frequently mentioned and rated as the best product in that category
+- On track for the largest market share among ICPs
+- Robust and reliable; major bugs rarely make it to production
+
+To advance from PMF to best-in-class:
+- Pay attention to small details
+- Innovate with major new feature bets
+- Conduct deep customer interviews to explore problems and find novel solutions
```

Comment:
I think there's something around scaling (though this might also be for PMF stage): "Our biggest customers can use this product at scale"